packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/index.js

```diff
@@ -146,3 +151,43 @@ function NavigationMenuEditor( { navigationMenu } ) {
 		</BlockEditorProvider>
 	);
 }
+
+const POPOVER_PROPS = {
+	position: 'bottom right',
+	variant: 'toolbar',
+};
+
+function ScreenNavigationMoreMenu( navigationMenuID ) {
+	const { deleteEntityRecord } = useDispatch( coreStore );
+	return (
+		<DropdownMenu
+			className="sidebar-navigation__more-menu"
+			icon={ moreVertical }
+			popoverProps={ POPOVER_PROPS }
+		>
+			{ ( { onClose } ) => (
+				<div>
+					<MenuGroup>
+						<MenuItem>{ __( 'Rename' ) }</MenuItem>
+						<MenuItem>{ __( 'Duplicate' ) }</MenuItem>
+						<MenuItem
+							isDestructive
+							isTertiary
+							onClick={ () => {
+								deleteEntityRecord(
+									'postType',
+									'wp_navigation',
+									navigationMenuID,
+									{ force: true }
+								);
+								onClose();
```

Comment:
Need to change this to go back to the previous screen, rather than stay in the deleted nav screen